api-extractor/src/DocItemLoader.ts

```diff
@@ -132,8 +140,8 @@ export default class DocItemLoader {
       }
     );
 
-    // JSON schema has been found and validated
-    this.cache[cachePackageName] = apiPackage;
+    const packageName: string = path.basename(packageJsonFilePath).split('.').shift();
```

Comment:
What if the filename has no extension?  Better to use path.parse() than write your own path parsing logic